internal/config/config_test.go

```diff
@@ -387,3 +331,59 @@ func TestMissingServerTLS(t *testing.T) {
 	assert.Equal(t, expected.Auth, result.Auth)
 	assert.Nil(t, result.TLS)
 }
+
+func getAgentConfig() *Config {
```

Comment:
Should we be using the GetAgentConfig function in `test/type/config.go` here?

Reply:
There is a circular dependency when using it here so I left this here